src/Controller/OcrController.php

```diff
@@ -177,8 +178,40 @@ public function homeAction(): Response
     }
 
     /**
-     * @Route("/api", name="api")
-     * @Route("/api.php", name="apiPhp")
+     * @Route("/api", name="api", methods={"GET", "POST"})
+     * @Route("/api.php", name="apiPhp", methods={"GET", "POST"})
+     * @OA\Parameter(
+     *     name="engine",
+     *     in="query",
+     *     description="The engine to use, either `tesseract` or `google`.",
+     *     @OA\Schema(type="string")
+     * )
+     * @OA\Parameter(
+     *     name="image",
+     *     in="query",
+     *     description="The image URL.",
+     *     @OA\Schema(type="string")
+     * )
+     * @OA\Parameter(
+     *     name="langs",
+     *     in="query",
+     *     description="List of language codes.
+     *         Can be left empty, in which case the engine will do its best (useful for unsupported languages).",
+     *     @OA\JsonContent(type="array", @OA\Items(type="string"))
+     * )
+     * @OA\Parameter(
+     *     name="psm",
+     *     in="query",
+     *     description="The Page Segmentation Mode for Tesseract.",
+     *     @OA\Schema(type="int")
+     * )
+     * @OA\Parameter(
```

Comment:
Noting that this one doesn't have an interactive input at all, but I guess it's not important.